### ISSUE TITLE
The loss

### ISSUE THREAD

**Issue by stvsd1314** [HoldenB/TD3-BC-from-scratch#1]
Hello,  want to know if this loss function "-lmbda * Q.mean() + F.mse_loss(pi, action)" setup is reasonable, because when Q is greater than 0, the TD3 loss term becomes a constant, which essentially degenerates into Behavior Cloning (BC), right?

**Comment by holdenb** [comment]
Hi, it's been a little while since I've looked back at the original paper, however if you see section 5 (page 6) I believe that will answer your question in more detail. The behavior cloning is a regularization term and highly depends on Q. It essentially becomes a balance between RL and imitation learning (which is what you're referring to by stating that it degenerates into behavior cloning).
